examples/README.md

```diff
@@ -0,0 +1,7 @@
+# KPT Function Catalog
```

Comment:
It's OK to check this in for now.
I want to understand the long term plan. We will eventually generate `examples/README.md` in GH action and no longer need to check in this file, right?

Reply:
That's correct; we can also remove it here.

Reply:
Great! Then let's remove it here